requirements.txt

```diff
@@ -14,3 +14,4 @@ python-dateutil == 2.7.5
 pytz == 2018.7
 requests == 2.20.0
 django-csp
+lxml
```

Comment:
Dette er diverre feil. Og det er fordi repoet er rotete.

Den requirements-fila er i bruk for `fkweb` og `infra`, men ikkje utils (som `fkupload` og `move_and_process` (`fkprocess`? :P)).

Staden du legg dette til (slik det er), er i `roles/upload-app/tasks/main.yml`.